Dockerfile

```diff
@@ -0,0 +1,37 @@
+FROM centos:centos7
+
+EXPOSE 8080
+
+ADD sendEmail-epel-7.repo /etc/yum.repos.d/
+
+RUN yum install -y epel-release && \
+    yum -y --setopt=tsflags=nodocs update && \
+    yum -y --setopt=tsflags=nodocs install httpd \
+                                           nagios \
+                                           telnet \
+                                           supervisor \
+                                           python-jinja2 \
+                                           nagios-plugins-all \
+                                           sendEmail \
+                                           perl-Net-SSLeay \
+                                           perl-IO-Socket-SSL && \
```

Comment:
Adding the `rpm -V` line as I mentioned in https://github.com/feedhenry/nagios3-container/pull/2/files/e779ee711ceb759ddd3ffb7c9c8ce3ac3ca0404c#r70609570 prevents us from building bad images (accidentally missing packages).